plugin/path_role.go

```diff
@@ -576,6 +617,19 @@ func (b *GcpAuthBackend) storeRole(ctx context.Context, s logical.Storage, roleN
 		return logical.ErrorResponse(err.Error()), nil
 	}
 
+	// Set default alias names
+	if role.IAMAliasType == "" {
+		role.IAMAliasType = defaultIAMAlias
+	}
+	if role.GCEAliasType == "" {
```

Comment:
This wouldn't be a backwards compatible change as described. An empty string needs to be interpreted as `unique_id` or `instance_id` otherwise the behavior will change for existing users. That being said, I'm okay with using a different value to not create an alias. Something like `noalias` or `no-alias`. I don't know the backstory as to why this is always creating an alias with no ability to not create an alias.

Reply:
As far as I know, only the token auth method doesn't result in any alias. I'm not sure what the presumptions are of an alias always being set up either, though it would be worth a discussion about the use cases for no alias. It would be a significant change that we'd want to review, and would be larger than a minor update.

Reply:
That's cool! Thanks for weighing in!